dlpy/tests/test_gpu.py

```diff
@@ -0,0 +1,539 @@
+        '''
+        gpu=Gpu(precision='FP16') in Model.fit() and Model.predict().
+            precision : string, optional
+                Specifies the experimental option to incorporate lower computational
+                precision in forward-backward computations to potentially engage tensor cores.
+                Valid Values: FP32, FP16
+                Default: FP32
+        '''
+        model1 = self.CreateSimpleCNN1()
+        
+        caslib, tmp_caslib, data, inputs, target = self.LoadEEE()
+
+        # Default lr=1e-2 causes FPE if precision='FP16'.
+        r = model1.fit(data=data, inputs=inputs, target=target, n_threads=16, mini_batch_size=8, lr=1e-4, 
+                gpu=Gpu(precision='FP16'))
+        self.assertTrue(r.severity == 0)
+
+        r2 = model1.predict(data=data, gpu=Gpu(precision='FP16'))
+        self.assertTrue(r2.severity == 0)
+
+        if (caslib is not None) and tmp_caslib:
+            self.s.retrieve('table.dropcaslib', message_level='error', caslib=caslib)
+                
+    def test_gpu05(self):
+        '''
+        gpu=Gpu(use_exclusive=True) in Model.fit() and Model.predict().
+            use_exclusive : bool, optional
+                Specifies exclusive use of GPU devices.
+                Default: False
+        '''
+        model1 = self.CreateSimpleCNN1()
+        
+        caslib, tmp_caslib, data, inputs, target = self.LoadEEE()
+
+        r = model1.fit(data=data, inputs=inputs, target=target, n_threads=16, mini_batch_size=8, lr=1e-4, 
+                gpu=Gpu(use_exclusive=True))
+        self.assertTrue(r.severity == 0)
+
+        r2 = model1.predict(data=data, gpu=Gpu(use_exclusive=True))
+        self.assertTrue(r2.severity == 0)
+
+        if (caslib is not None) and tmp_caslib:
+            self.s.retrieve('table.dropcaslib', message_level='error', caslib=caslib)
+
+    def test_gpu06(self):
+        '''
+        gpu=Gpu(devices=[0], use_tensor_rt=True) in Model.fit() and Model.predict().
+            devices : list-of-ints, optional
+                Specifies a list of GPU devices to be used.
+            use_tensor_rt : bool, optional
+                Enables using TensorRT for fast inference.
+                Default: False.
+        '''
+        model1 = self.CreateSimpleCNN1()
+        
+        caslib, tmp_caslib, data, inputs, target = self.LoadEEE()
+
+        r = model1.fit(data=data, inputs=inputs, target=target, n_threads=16, mini_batch_size=8, lr=1e-4, 
+                gpu=Gpu(devices=[0], use_tensor_rt=True))
+        #'WARNING: TensorRT only supports inference and will be disabled.'
+        self.assertTrue(r.severity == 1)
+
+        r2 = model1.predict(data=data, gpu=Gpu(devices=[0], use_tensor_rt=True))
+        self.assertTrue(r2.severity == 0)
+
+        if (caslib is not None) and tmp_caslib:
+            self.s.retrieve('table.dropcaslib', message_level='error', caslib=caslib)
+
+    # Skip the test until Alireza fixes TensorRT issues.
```

Comment:
Please remove.  If this unit test will not run as-is, please simply comment out.